### DISCUSSION TITLE
Barricade server-side optimization.

### DISCUSSION THREAD

**Discussion by DrDallas1** [SmartlyDressedGames/Unturned-3.x-Community#3704] [General]
Hello,

Are there any plans for optimizing performance for server owners that have servers with large amounts of barricades. If the server has a particularly large player count, vehicle and barricade calculations make the tick rate slow to a crawl, and either require extreme limitations on barricades or give extreme performance downsides.

Thank you.

**Comment by SDGNelson** [comment]
One relevant planned optimization is: #3011
